public-api/build.gradle

```diff
@@ -434,7 +455,13 @@ test {
     maxHeapSize = '2048m'
 }
 
+def cloudSdkHomePath = System.getenv('CIRCLECI') == 'true' ?  "/home/circleci/google-cloud-sdk" : "/google-cloud-sdk"
+
+
 appengine {  // App Engine tasks configuration
+    tools {
+        cloudSdkHome = cloudSdkHomePath
+    }
```

Comment:
RWB has a line here `cloudSdkVersion = '392.0.0'` - not sure if that is helpful for you as well